sda-commons-server-dropwizard/README.md

```diff
@@ -86,6 +86,48 @@ public void initialize(Bootstrap<Configuration> bootstrap) {
 }
 ```
 
+### MetadataContextBundle
```

Comment:
Good technical documentation! 🏎️ 

We could also add a paragraph "What can I do with the metadata context" from a business perspective.

Reply:
Yes, in a follow up.